pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>com.github.mpkorstanje</groupId>
 	<artifactId>simmetrics</artifactId>
-	<version>5.0.0-SNAPSHOT</version>
+	<version>4.1.1-klara.1-SNAPSHOT</version>
```

Comment:
I expected a format that equals the versioning we did in ignite and avro and ...

`4.1.1.klara.1-SNAPSHOT`

Continue with dot, instead of dash.